Cargo.toml

```diff
@@ -17,9 +17,3 @@ panic = "abort"
 
 [replace]
 "https://github.com/rust-lang/crates.io-index#serde_derive:1.0.80" = { git = "https://github.com/servo/serde", branch = "deserialize_from_enums9", feature = "deserialize_in_place" }
```

Comment:
I think we no longer need `[replace]` here, and we can use `[patch]` again: https://github.com/szeged/webrender/blob/721181711e8208bf3c57ef7fb76d8299e2b2d8ee/Cargo.toml#L15-L16